nuget/OpenTelemetry.AutoInstrumentation/contentFiles/any/any/instrument.cmd

```diff
@@ -1,13 +1,42 @@
 @echo off
-setlocal
+setlocal ENABLEDELAYEDEXPANSION
 
 :: This script is expected to be used in a build that specified a RuntimeIdentifier (RID)
 set BASE_PATH=%~dp0
+set COR_PROFILER_PATH=%BASE_PATH%OpenTelemetry.AutoInstrumentation.Native.dll
+
+:: Validate
+IF EXIST %COR_PROFILER_PATH% (
+    set CORECLR_PROFILER_PATH=!COR_PROFILER_PATH!
+) ELSE (
+    set "COR_PROFILER_PATH="
+    echo RuntimeIdentifier was not specified when building the project. Attempting to use the fallback location.
+
+    set COR_PROFILER_PATH_64=%BASE_PATH%runtimes\win-x64\native\OpenTelemetry.AutoInstrumentation.Native.dll
```

Comment:
Nice, this should work for `dotnet build`.

Reply:
maybe "Attempting to use Windows x64 for fallback" to give more information, what is the fallback.